packages/components/src/card/styles/card-styles.js

```diff
@@ -3,12 +3,18 @@
  */
 import styled from '@emotion/styled';
 
+/**
+ * WordPress dependencies
+ */
+import { __experimentalPrimitives } from '@wordpress/icons';
```

Comment:
I moved the primitives to the icons package to avoid the double dependency between components and icons.
In a separate PR, these should be in their own package `@wordpress/primitives`